lib/rubygems/commands/update_command.rb

```diff
@@ -97,8 +97,10 @@ def execute
     if options[:explain]
       say "Gems to update:"
 
-      gems_to_update.each do |(name, version)|
-        say "  #{name}-#{version}"
+      gems_to_update.each do |(name, version, platform)|
+        say platform == "ruby" ?
```

Comment:
should the explanation here be the same as when installing?

Reply:
Makes sense, I'll try to reconcile that :+1:

Reply:
I reviewed this. The logic is, although similar, not the same. But the list has the same shape I think. Since there are other fixes coming to this command (number 1 in https://github.com/rubygems/rubygems/issues/2629), I think it's better not to spend time on trying to DRY this up until I see how the final version with all fixes looks like.

Reply:
Ok, I have a refactor of this logic, but I would prefer to add it on a separate PR since it affects the install command as well, and it's not relevant to this bug fix. Is it ok?

Reply:
ok, sounds good